src/main/java/ch/njol/skript/expressions/ExprMessage.java

```diff
@@ -63,19 +61,22 @@
 		"	set quit message to \"%player% left this awesome server!\"",
 		"",
 		"on death:",
-		"	set the death message to \"%player% died!\""})
-@Since("1.4.6 (chat message), 1.4.9 (join & quit messages), 2.0 (death message)")
+		"	set the death message to \"%player% died!\"",
```

Comment:
The whole Example annotation lines are not using tabs correctly, replace it with `\t`

Reply:
Existing implementation

Reply:
I think it's okay to edit it while you're at it, it's not much of work.